### ISSUE TITLE
Solution

### ISSUE THREAD

**Issue by torarnv** [AdnanHodzic/synology-znc-autostart#1]
Another solution is just to run

```
sudo su
/var/packages/znc/scripts/start-stop-status start
```

And let it create the .pem file. Then you can stop it and start it from the Synology Web UI

**Comment by AdnanHodzic** [comment]
@torarnv thanks, I just did this, and I can verify that it really starts ZNC via DSM and persists across reboots. 

Thanks for making my script obsolete :P

**Comment by AdnanHodzic** [comment]
@torarnv after one of the latest updates, I've noticed that after reboots ZNC wouldn't start at boot again. 

Of course, this time I could start it using "Package Center", but it was still annoying having to do this after every reboot. 

I've just made update to the script, by replacing run which was made from user custom ZNC binary `/volume1/@appstore/znc/bin/znc` to run `/var/packages/znc/scripts/start-stop-status start` instead. 

Which will run ZNC binary from standard (Package Center) location:
 `/usr/local/znc/bin/znc -d /usr/local/znc/var` as "znc" user.

Thought you might be interested.
